RELEASE.md

```diff
@@ -14,8 +22,17 @@ make test-race
 go mod tidy
 git status # should be no changes
 
-# Start mev-boost with relay check, and call the mev-boost status endpoint
+# Start mev-boost with relay check and -relays
```

Comment:
I'd prefer a more verbose explanation of what -relay-check and -relays does here (if it is not already explicitly defined). Or, a redirect to check the cli section for more details. *Noticed that -relays is not mentioned in the cli section, by the way. 

"Starts mev-boost on mainnet. `-relay-check `checks relay status on startup and when status API is called. 
.. etc"

OR:

"Check out mev-boost cli arguments for more details"

Reply:
That's details for the readme, not for this release checklist